modules/ROOT/pages/workflow.adoc

```diff
@@ -2,10 +2,6 @@
 
 AeroGear Mobile Services provide features described in the section about xref:con_services.adoc[Mobile Services].
 
-The workflow for connecting your mobile app to mobile services can be summarized as:
-
-image::3steps.png[]
-
 . xref:provisioning-services.adoc[Provision and configure a mobile service]
```

Comment:
these won't work, but can merge and follow up? as long as @johnfriz  is happy with diagram

Reply:
@finp - can you clarify what you mean by "these won't work"?

Reply:
not any of the anointed urls, wanna replace everything under the png  with:

The following Mobile Services are available:

* xref:identity-management.adoc[{keycloak-service}]	
* xref:push-notifications.adoc[{unifiedpush-service}]	
* xref:runtime-connector.adoc[{custom-service}]	
* xref:mobile-metrics.adoc[{metrics-service}]	
* xref:mobile-cicd.adoc[{digger-service}]	
* xref:device-security.adoc[{device-security-service}]

Reply:
@finp - merge away.